class DataCollectorError(Exception):
    """Custom exception for DataCleaner-related errors."""


class DataCleanerError(Exception):
    """Custom exception for DataCleaner-related errors."""